LICENSE

```diff
@@ -199,3 +199,86 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+***************************************************************************************************
```

Comment:
I just want the license to have a title: so please remove 203-212 and add a single line with the name of the license: "CC0 1.0 Universal License" and a line of ==== to make it stand out for someone looking in the LICENSE file.